src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Base, AaveV3BaseEModes} from 'aave-address-book/AaveV3Base.sol';
+import {AaveV3PayloadBase} from 'aave-helpers/v3-config-engine/AaveV3PayloadBase.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+/**
+ * @title weETH Aave V3 Base Onboarding
+ * @author ACI
+ * - Snapshot: TODO
+ * - Discussion: https://governance.aave.com/t/arfc-onboarding-of-weeth-to-aave-v3-on-base/17691
+ */
+contract AaveV3Base_WeETHAaveV3BaseOnboarding_20240527 is AaveV3PayloadBase {
+  using SafeERC20 for IERC20;
+
+  address public constant weETH = 0x04C0599Ae5A44757c0af6F9eC3b93da8976c150A;
+  uint256 public constant weETH_SEED_AMOUNT = 0.001 ether;
+
+  function _postExecute() internal override {
+    IERC20(weETH).forceApprove(address(AaveV3Base.POOL), weETH_SEED_AMOUNT);
+    AaveV3Base.POOL.supply(weETH, weETH_SEED_AMOUNT, address(AaveV3Base.COLLECTOR), 0);
+  }
+
+  function newListings() public pure override returns (IAaveV3ConfigEngine.Listing[] memory) {
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: weETH,
+      assetSymbol: 'weETH',
+      priceFeed: 0xFc4d1d7a8FD1E6719e361e16044b460737F12C44,
+      eModeCategory: AaveV3BaseEModes.ETH_CORRELATED,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 72_50,
+      liqThreshold: 75_00,
+      liqBonus: 7_50,
+      reserveFactor: 45_00,
+      supplyCap: 150,
+      borrowCap: 30,
+      debtCeiling: 0,
+      liqProtocolFee: 10_00,
+      rateStrategyParams: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(35_00),
```

Comment:
45%

Reply:
https://governance.aave.com/t/arfc-adjusting-interest-rate-curve-for-weeth-on-arbitrum-and-base/17804